index.css

```diff
@@ -0,0 +1,446 @@
+@import url("https://fonts.googleapis.com/css2?family=PT+Sans+Narrow:wght@400;700&display=swap");
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+  overflow-x: hidden;
+}
+
+body {
+  font-family: "PT Sans Narrow", sans-serif;
+  background-color: var(--body-bg-color);
+}
+
+h1 {
+  color: var(--ghost-color);
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.home-sub-heading {
+  font-family: Jurassic Park;
+  letter-spacing: 5px;
+  word-spacing: 0.5rem;
+  font-size: 4.5rem;
+  color: var(--font-color);
+  flex-basis: 100%;
+}
+
+.home-screen {
+  display: flex;
+  flex-direction: column;
```

Comment:
I see you are repeating these styles in multiple places consider making utility class for it.
```html
  display: flex;
  flex-direction: column;
```